tests/conftest.py

```diff
@@ -298,6 +307,50 @@ def email_target_json():
     return targets
 
 
+@pytest.fixture
+def multiple_click_object():
+    """Return a list of clicks to match the correct number of unique users."""
+    clicks = list()
+    for x in range(1, 2):
+        clicks.append(
+            Click(
+                message="Testing",
+                user="jane.smith1@domain.tld",
+                source_ip="8.8.8.8",
+                time="01/01/2025 13:00",
+                application="NA",
+            )
+        )
+        clicks.append(
+            Click(
+                message="Testing",
+                user="john.doe1@domain.tld",
+                source_ip="8.8.8.8",
+                time="01/01/2025 13:00",
+                application="NA",
+            )
+        )
+        clicks.append(
+            Click(
+                message="Testing",
+                user="jane.smith2@domain.tld",
+                source_ip="8.8.8.8",
+                time="01/01/2025 13:00",
+                application="NA",
+            )
+        )
+        clicks.append(
+            Click(
+                message="Testing",
+                user="john.doe2@domain.tld",
+                source_ip="8.8.8.8",
```

Comment:
Avoid using public IP addresses in test code when possible:
```suggestion
                source_ip="10.0.0.3",
```

Reply:
Changed in f5ee8f2